### PR TITLE
AVX-20480: Update GCP vpc_id to include project name

### PR DESCRIPTION
**Pull request by mlin-aviatrix**

### CODE REVIEW

File: docs/resources/aviatrix_gateway.md [hunk 1]
```diff
@@ -287,7 +287,11 @@ The following arguments are supported:
 !> When creating a Gateway with an Azure VNet created in Controller version 6.4 or earlier or with an Azure VNet created out of band, referencing `vpc_id` in another resource on the same apply that creates this Gateway will cause Terraform to throw an error. Please use the Gateway data source to reference the `vpc_id` of this Gateway in other resources.
 
 ~> As of Provider version R2.21.2+, the `vpc_id` of an OCI VCN has been changed from its name to its OCID.
-* `vpc_id` - (Required) VPC ID/VNet name of cloud provider. Example: AWS/AWSGov/AWSChina: "vpc-abcd1234", GCP: "vpc-gcp-test", Azure/AzureGov/AzureChina: "vnet_name:rg_name:resource_guid", OCI: "ocid1.vcn.oc1.iad.aaaaaaaaba3pv6wkcr4jqae5f44n2b2m2yt2j6rx32uzr4h25vqstifsfdsq".
+
+-> As of Provider version R.22.0+, the `vpc_id` of a GCP VPC has been updated to include the project name.
+
+-> As of Provider version R.22.0+, the `vpc_id` of a GCP VPC has been updated to include the project name.
```
Comment (inc1t3Ful) [inline docs/resources/aviatrix_gateway.md]: Is this a typo with duplicate? 
I think also should be a `~>`

File: docs/resources/aviatrix_spoke_gateway.md [hunk 2]
```diff
@@ -234,7 +234,9 @@ The following arguments are supported:
 !> When creating a Spoke Gateway with an Azure VNet created in Controller version 6.4 or earlier or with an Azure VNet created out of band, referencing `vpc_id` in another resource on the same apply that creates this Spoke Gateway will cause Terraform to throw an error. Please use the Spoke Gateway data source to reference the `vpc_id` of this Spoke Gateway in other resources.
 
 ~> As of Provider version R2.21.2+, the `vpc_id` of an OCI VCN has been changed from its name to its OCID.
-* `vpc_id` - (Required) VPC-ID/VNet-Name of cloud provider. Example: AWS/AWSGov/AWSChina: "vpc-abcd1234", GCP: "vpc-gcp-test", Azure/AzureGov/AzureChina: "vnet_name:rg_name:resource_guid", OCI: "ocid1.vcn.oc1.iad.aaaaaaaaba3pv6wkcr4jqae5f44n2b2m2yt2j6rx32uzr4h25vqstifsfdsq".
+
+-> As of Provider version R.22.0+, the `vpc_id` of a GCP VPC has been updated to include the project name.
```
Comment (inc1t3Ful) [inline docs/resources/aviatrix_spoke_gateway.md]: see comment above about the type of note to leave

File: docs/resources/aviatrix_transit_gateway.md [hunk 3]
```diff
@@ -290,7 +290,9 @@ The following arguments are supported:
 !> When creating a Transit Gateway with an Azure VNet created in Controller version 6.4 or earlier or with an Azure VNet created out of band, referencing `vpc_id` in anothe resource on the same apply that creates this Transit Gateway will cause Terraform to throw an error. Please use the Transit Gateway data source to reference the `vpc_id` of this Transit Gateway in other resources.
 
 ~> As of Provider version R2.21.2+, the `vpc_id` of an OCI VCN has been changed from its name to its OCID.
-* `vpc_id` - (Required) VPC-ID/VNet-Name of cloud provider. Example: AWS/AWSGov/AWSChina: "vpc-abcd1234", GCP: "vpc-gcp-test", Azure/AzureGov/AzureChina: "vnet_name:rg_name:resource_guid", OCI: "ocid1.vcn.oc1.iad.aaaaaaaaba3pv6wkcr4jqae5f44n2b2m2yt2j6rx32uzr4h25vqstifsfdsq".
+
+-> As of Provider version R.22.0+, the `vpc_id` of a GCP VPC has been updated to include the project name.
```
Comment (inc1t3Ful) [inline docs/resources/aviatrix_transit_gateway.md]: * see above

File: docs/resources/aviatrix_vpc.md [hunk 4]
```diff
@@ -158,6 +158,8 @@ In addition to all arguments above, the following attributes are exported:
 -> **NOTE** `vpc_id` should only be referenced to create Gateways, Spoke Gateways and Transit Gateways. The vpc_id attribute of the gateway created should be referenced to create other resources such as VPN Users or Site2Cloud connections.
 
 ~> As of Provider version R2.21.2+, the `vpc_id` of an OCI VCN has been changed from its name to its OCID.
+
+-> As of Provider version R.22.0+, the `vpc_id` of a GCP VPC has been updated to include the project name.
```
Comment (inc1t3Ful) [inline docs/resources/aviatrix_vpc.md]: * see above

**Review by inc1t3Ful — APPROVED** [review]

**Review by zhiqiangjin — APPROVED** [review]

**Pull request merged**